src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for the case when 'Roboto' is not available. For example, 'font-family: Roboto, Arial, sans-serif;' to ensure text is still readable with a default font.